source/backend/api/Services/AcquisitionFileService.cs

```diff
@@ -345,21 +337,14 @@ public PimsAcquisitionFile UpdateProperties(PimsAcquisitionFile acquisitionFile,
                     throw new BusinessRuleViolationException("You must remove all takes and interest holders from an acquisition file property before removing that property from an acquisition file");
                 }
                 _acquisitionFilePropertyRepository.Delete(deletedProperty);
-                /*
-                TODO: Fix mapings
-                if (deletedProperty.Property.IsPropertyOfInterest)
+
+                var totalAssociationCount = _propertyRepository.GetAllAssociationsCountById(deletedProperty.PropertyId);
```

Comment:
Please leave a note for Sue/Praveen about this, this doesn't seem directly related to the story.